ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -90,240 +93,154 @@ export class Send extends IronfishCommand {
-      this.log(`Raw transaction: ${rawTransactionResponse}`)
-      this.log(`\nRun "ironfish wallet:post" to post the raw transaction. `)
+      this.log('Raw Transaction')
+      this.log(RawTransactionSerde.serialize(raw).toString('hex'))
+      this.log(`Run "ironfish wallet:post" to post the raw transaction. `)
       this.exit(0)
     }
 
-    // Run the progress bar for about 2 minutes
-    // Chances are that the transaction will finish faster (error or faster computer)
-    const bar = CliUx.ux.progress({
-      barCompleteChar: '\u2588',
-      barIncompleteChar: '\u2591',
-      format: 'Creating the transaction: [{bar}] {percentage}% | ETA: {eta}s',
-    }) as ProgressBar
-
-    bar.start()
-
-    let value = 0
-    const timer = setInterval(() => {
-      value++
-      bar.update(value)
-      if (value >= bar.getTotal()) {
-        bar.stop()
-      }
-    }, 1000)
-
-    const stopProgressBar = () => {
-      clearInterval(timer)
-      bar.update(100)
-      bar.stop()
+    if (!flags.confirm && !(await this.confirm(assetId, amount, raw.fee, from, to))) {
+      this.error('Transaction aborted.')
     }
 
-    let transaction
+    CliUx.ux.action.start('Sending the transaction')
 
-    try {
-      const result = await client.postTransaction({
-        transaction: rawTransactionResponse,
-        account: from,
-      })
-
-      stopProgressBar()
+    const response = await client.postTransaction({
+      transaction: RawTransactionSerde.serialize(raw).toString('hex'),
+      account: from,
+    })
 
-      const transactionBytes = Buffer.from(result.content.transaction, 'hex')
-      transaction = new Transaction(transactionBytes)
+    const bytes = Buffer.from(response.content.transaction, 'hex')
+    const transaction = new Transaction(bytes)
 
-      this.log(`
-Sending ${CurrencyUtils.renderIron(amount, true, assetId)} to ${to} from ${from}
-Transaction Hash: ${transaction.hash().toString('hex')}
-Transaction fee: ${CurrencyUtils.renderIron(transaction.fee(), true)}
+    CliUx.ux.action.stop()
 
-Find the transaction on https://explorer.ironfish.network/transaction/${transaction
+    this.log(`Sent ${CurrencyUtils.renderIron(amount, true, assetId)} to ${to} from ${from}`)
+    this.log(`Hash: ${transaction.hash().toString('hex')}`)
+    this.log(`Fee: ${CurrencyUtils.renderIron(transaction.fee(), true)}`)
+    this.log(
+      `\nIf the transaction is mined, it will appear here https://explorer.ironfish.network/transaction/${transaction
         .hash()
-        .toString(
-          'hex',
-        )} (it can take a few minutes before the transaction appears in the Explorer)`)
-    } catch (error: unknown) {
-      stopProgressBar()
-      this.log(`An error occurred while sending the transaction.`)
-      if (error instanceof Error) {
-        this.error(error.message)
-      }
-      this.exit(2)
-    }
+        .toString('hex')}`,
+    )
 
-    if (flags.watch && transaction) {
+    if (flags.watch && params) {
```

Comment:
nit - do we need `&& params`? Won't it always be truthy?

Reply:
fixed